src/main/features/directions-questionnaire/views/hearing-location.njk

```diff
@@ -52,20 +52,20 @@
                 {
                   label: 'Yes',
                   value: YesNoOption.YES.option,
-                  checked: form.valueFor('courtAccepted') === YesNoOption.YES
+                  checked: form.valueFor('courtAccepted').option === YesNoOption.YES.option
                 },
                 {
                   label: 'No',
                   value: YesNoOption.NO.option,
-                  checked: form.valueFor('courtAccepted') === YesNoOption.NO
+                  checked: form.valueFor('courtAccepted').option === YesNoOption.NO.option
                 }
               ]
             ) }}
             <div class="js-hidden" id="courtAccepted-no">
               <h1 class="heading-medium">{{ t('Choose another court') }}</h1>
               {{ radioOption(form = form, label = 'Enter a court name', name = 'alternativeOption', value = 'name', checked = form.valueFor('alternativeOption') === 'name', bold = true) }}
               <div class="form-group panel panel-border-narrow js-hidden " id="alternativeOption-name" aria-hidden="false">
-                {{ textInput(label = 'Court name', form = form, name = 'alternativeCourtName', inputClass = 'full-width') }}
+                {{ textInput(label = 'Court name', form = form, name = 'alternativeCourtName', inputClass = 'full-width') }}Do you want to choose
```

Comment:
this looks like a copy/paste gone bad. Please remove

Reply:
done